doc/develop/test/twister.rst

```diff
@@ -699,6 +699,21 @@ required_snippets: <list of needed snippets>
               - cdc-acm-console
               - user-snippet-example
 
+bsim_copy_exe: <True|False> (default False)
+    If true, twister will try to copy executable file (`zephyr.exe`) to
+    BabbleSim `bin` directory (`${BSIM_OUT_PATH}/bin`), when building is
+    finished. This action is necessary to make it possible to execute BabbleSim
+    tests. By default, the name of copied executable is as following (with dots
+    and slashes replaced by underscores):
+    `bs_<platform_name>_<test_path>_<test_scenario_name>`.
+    This name can be changed by setting `bsim_exe_name` option.
+
+bsim_exe_name: <string>
+    If provided, twister will use it during copying executable file to BabbleSim
```

Comment:
Maybe better:
"If provided, the executable filename when copying to BabbleSim's bin directory, 
will be `bs_<platform_name>_<bsim_exe_name>` instead of the default based on the
test path and scenario name"